src/components/editor/customCompleter.js

```diff
@@ -192,6 +192,44 @@ export const customCompleter = {
             "yellowgreen"
         ];
 
+        let Texture = [
+            "bark",
+            "blank", 
+            "brick",
+            "cement",
+            "cobblestone",
+            "duck",
+            "error",
+            "floral",
+            "grass",
+            "ground",
+            "illusion",
+            "lava",
+            "maze",
+            "metal",
+            "myr",
+            "night_sky",
+            "patterned_glass",
+            "pavement",
+            "sand",
+            "snow",
+            "steel",
+            "stars",
+            "water",
+            "wicker",
+            "wood",
+            "wool"
+        ];
```

Comment:
You should never repeat yourself in code. These should be pulled programmatically from the same place where they're used for the textures.